common/PacketTypes.asn1

```diff
@@ -0,0 +1,29 @@
+PacketTypes DEFINITIONS ::= BEGIN
```

Comment:
it should go into separate directory - ccsds is a little bit "outside" PUS standards